specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

```diff
@@ -1515,6 +1515,24 @@
         }
       }
     },
+    "RegistryIdentity": {
```

Comment:
@xiadu94 
* This new type is not referenced anywhere else in the model.. what is its purpose?
  * If this is indeed an object created at the server side and returned to the user, you should mark the properties as readonly. example [here](https://github.com/Azure/azure-rest-api-specs/blob/198ac8b56e48dd29e0f907282e01bb0da4eeed06/specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json#L1483)
* Is this meant to be returned from the server side or is it a config object that the user will create?